src/b2/DirectDiscImage.cpp

```diff
@@ -6,12 +6,18 @@
 #include "native_ui.h"
 #include "Messages.h"
 #include <limits.h>
+#ifndef B2_LIBRETRO_CORE
 #include "load_save.h"
+#else
+#include "../libretro/adapters.h"
+#endif // B2_LIBRETRO_CORE
 
 //////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////
 
+#ifndef B2_LIBRETRO_CORE
```

Comment:
You can have `LOG_DEFINE(OUTPUT,"OUTPUT",&log_printer_nowhere,false);` in core.cpp and then the logging can stay in too. The `false` is the state of the enable flag, which is tested by the `LOGF` macro directly before doing the format string.

Reply:
Thanks, reverted.